enterprise/models.py

```diff
@@ -2304,3 +2302,81 @@ def __repr__(self):
         Return uniquely identifying string representation.
         """
         return self.__str__()
+
+
+class PendingEnterpriseCustomerAdminUser(TimeStampedModel):
+    # pylint: disable=line-too-long
+    """
+    Model for pending enterprise admin users.
+
+    .. pii: The user_email field contains PII, but locally deleted via enterprise.signals.assign_or_delete_enterprise_admin_role when the admin registers a new account.
+    .. pii_types: email_address
+    .. pii_retirement: local_api, consumer_api
+    """  # pylint: enable=line-too-long
+
+    enterprise_customer = models.ForeignKey(
+        EnterpriseCustomer,
+        blank=False,
+        null=False,
+        on_delete=models.CASCADE,
+    )
+    user_email = models.EmailField(null=False, blank=False, unique=True)
```

Comment:
Do you want to remove the `unique=True` from here since we're doing a unique_together on customer and email?

Reply:
Ah, yes. I meant to take that out when I added in the `unique_together` 😄 Good catch.